parse.y

```diff
@@ -149,22 +149,24 @@ literal_hash(VALUE a)
     return rb_iseq_cdhash_hash(a);
 }
 
-static int
-script_lines_defined(void)
+static ID
+script_lines(void)
 {
-    ID script_lines;
+    ID script_lines; \
```

Comment:
```suggestion
    ID script_lines;
```

Reply:
Thanks, fixed it in https://github.com/ruby/ruby/pull/7990/commits/83d67923f26bfa31c7e5502392cc943656ff7bb3